amnestywebsite/humanity-theme
# Decommissions the Background Media block

Ref: https://github.com/amnestywebsite/humanity-theme/issues/260

Removes everything related to the background media block and decommissions it

**Steps to test**:
1. Edit a post
2. Open the block inserter and search for the background media block
3. There should be no results
4. You can also try and add a background media block in the code editor, and it will not work

Example: http://bigbite.im/i/Uv9qjL
